Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceToTemplateInterface.h

```diff
@@ -46,7 +46,16 @@ namespace AzToolsFramework
             //! @return The string matching the path to the entity alias
             virtual AZStd::string GenerateEntityAliasPath(AZ::EntityId entityId) = 0;
 
-            virtual void AppendEntityAliasToPatchPaths(PrefabDom& providedPatch, AZ::EntityId entityId, AZStd::string prefix = "") = 0;
+            //! Generates an entity alias from given entity id and prefix and then add it as path into the provided patch.
+            //! @param providedPatch The patch to add entity alias path to.
+            //! @param entityId The entity id to use for generating alias path.
+            //! @param prefix The prefix that will be added in front of the generated entity alias from entityId. 
+            virtual void AppendEntityAliasToPatchPaths(PrefabDom& providedPatch, AZ::EntityId entityId, const AZStd::string& prefix = "") = 0;
+
+            //! Add given entity alias path into the provided patch.
+            //! @param providedPatch The patch to add entity alias path to.
+            //! @param entityAliasPath The entity alias path to be added as path into the patch.
+            virtual void AppendEntityAliasPathToPatchPaths(PrefabDom& providedPatch, const AZStd::string& entityAliasPath) = 0;
```

Comment:
I incline to keep it since we need to re-create and copy entity alias path over and over again, especially while generating update parent entity patch.